src/components/app/index.js

```diff
@@ -11,6 +11,8 @@ import Metronome from '../../utils/metronome';
 import { setActivePeer } from '../../utils/api/peers';
 import { accountUpdated } from '../../actions/account';
 import Dialog from '../dialog';
+// temporary
```

Comment:
It might be better to write these comments more descriptive, something like:

`// TODO: next line should be moved to XXX once we setup YYY.`

But maybe it's not worth in this case, depends on how soon do you expect it to be resolved.